stumpy/stump.py

```diff
@@ -551,6 +564,12 @@ def stump(T_A, m, T_B=None, ignore_trivial=True, normalize=True, p=2.0, k=1):
         when k > 1. If you have access to a GPU device, then you may be able to
         leverage `gpu_stump` for better performance and scalability.
 
+    T_A_subseq_isconstant : numpy.ndarray, default None
```

Comment:
It's probably a good idea to remind the user that this parameter is ignored when `normalize=False`.

Reply:
Right! Thanks for pointing this out. I will revise it.